docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst

```diff
@@ -0,0 +1,305 @@
+  Here, ``sample_id`` from the ``samples.tsv`` file will shuffle all samples, then split them between
+  train/validation/test sets.
+- ``split_dataset:train_fraction``: Fraction of the dataset's ``sample_id`` in the train set. In our case ``0.6``.
+- ``split_dataset:test_fraction``: Fraction of the dataset's ``sample_id`` in the test set. In our case ``0.1``.
+
+  .. code-block:: xml
+
+      "split_method" : "sample_id"
+      "train_fraction": 0.6
+      "test_fraction": 0.1
+
+- ``training_time:num_epochs``: The maximum number of epochs that will be run during training. Each epoch is composed
+  of a training part and an evaluation part. It should be a strictly positive integer. In our case, we will use
+  50 epochs.
+
+  .. code-block:: xml
+
+     "num_epochs": 50
+
+- ``default_model:length_2D``: Size of the 2D patches used as model’s input tensors. We recommend using patches
+  between 256x256 and 512x512. In our case, we use patches of 256x256.
+- ``default_model:stride_2D``: Pixels’ shift over the input matrix to create 2D patches. In our case, we use
+  a stride of 244 pixels in both dimensions, resulting in an overlap of 12 pixels between patches.
+
+  .. code-block:: xml
+
+     "length_2D": [256, 256]
+     "stride_2D": [244, 244]
+
+- ``postprocessing:binarize_maxpooling``: Used to binarize predictions across all classes in multiclass models.
+
+  .. code-block:: xml
+
+      "binarize_maxpooling": {}
+
+- ``transformation:Resample``: Used to resample images to a common resolution (in mm) before splitting into patches,
+  according to each image real pixel size. In our case, we resample the images to a common resolution of 0.0001 mm
+  (0.1 um) in both dimensions.
+
+  .. code-block:: xml
+
+     "Resample":
+        {
+            "hspace": 0.0001,
+            "wspace": 0.0001
+        },
+
+
+Train model
+-----------
+
+Once the configuration file is ready, run the training:
+
+.. code-block:: bash
+
+   ivadomed -c config_microscopy.json
+
+Alternatively, the "command", "path_output", and "path_data" arguments can be passed as CLI flags
+in which case they supersede the configration file, see `Usage <../usage.html>`__.
+
+.. code-block:: bash
+
+   ivadomed --train -c config_microscopy.json --path-data path/to/bids/data --path-output path/to/output/directory
+
+.. note::
+
+   If a `compatible GPU <https://pytorch.org/get-started/locally/>`_ is available, it will be used by default.
+   Otherwise, training will use the CPU, which will take a prohibitively long computational time (several hours).
+
+The main parameters of the training scheme and model will be displayed on the terminal, followed by the loss value
+on training and validation sets at every epoch. To know more about the meaning of each parameter, go to
+:doc:`../configuration_file`. The value of the loss should decrease during the training.
+
+.. code-block:: console
```

Comment:
We should probably update all our tutorials eventually (not the most important thing), but the output in the console doesn't match the output in our tutorials (regardless of the INFO part, some prints don't match): 
![image](https://user-images.githubusercontent.com/49137243/148982559-231097dc-d6eb-44c9-a64f-236114e1f3fb.png)

Reply:
It may be a mistake on my part when removing the "date/info" part of the output. I'll double-check.

Reply:
I did not find significant differences but made some minor aesthetic changes in 69b2901.